src/components/Nav/Nav.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import './Nav.scss';
+import { faCartShopping } from '@fortawesome/free-solid-svg-icons';
+import { Link } from 'react-router-dom';
+
+const Nav = () => {
+  return (
+    <nav className="nav">
+      <h1 className="logo">
+        <Link to="/" className="logoLink">
+          남바완
+        </Link>
+      </h1>
+      <ul className="pageMenu">
+        {PAGENAV_LIST.map(menu => {
```

Comment:
```suggestion
        {PAGENAV_LIST.map(({id, url, page}) => {
```
매개변수자리에서 구조분해 해줘도 좋겠네요!